### PR TITLE
Configure tide to automerge PRs on repos owned by our team

### PR DESCRIPTION
**Pull request by smukherj1**
bazel-toolchains: Automerge un-approved PRs generated by robot & approved PRs by everyone else.
rules_docker & rules_k8s: Automerge approved PRs

### CODE REVIEW

File: prow/config.yaml [hunk 1]
```diff
@@ -426,8 +426,33 @@ tide:
...
+    - automerge
+    missingLabels:
+    - do-not-merge
+    - do-not-merge/hold
+    - do-not-merge/invalid-owners-file
+    - do-not-merge/work-in-progress
+  - repos:
+    - bazelbuild/bazel-toolchains
+    - bazelbuild/rules_docker
+    - bazelbuild/rules_k8s
```
Comment (cjwagner) [inline prow/config.yaml]: Please combine this query with the query for `GoogleCloudPlatform/netd` to save API rate limit tokens.

Comment (smukherj1) [reply prow/config.yaml]: Done

**Comment by smukherj1** [comment]
Done. PTAL

**Pull request merged**